evadb/server/command_handler.py

```diff
@@ -23,11 +23,37 @@
 from evadb.optimizer.plan_generator import PlanGenerator
 from evadb.optimizer.statement_to_opr_converter import StatementToPlanConverter
 from evadb.parser.parser import Parser
+from evadb.parser.statement import AbstractStatement
 from evadb.parser.utils import SKIP_BINDER_AND_OPTIMIZER_STATEMENTS
 from evadb.utils.logging_manager import logger
 from evadb.utils.stats import Timer
 
 
+def execute_statement(
+    evadb: EvaDBDatabase,
+    stmt: AbstractStatement,
+    do_not_raise_exceptions: bool = False,
+    do_not_print_exceptions: bool = False,
+    **kwargs
+) -> Iterator[Batch]:
+    # For certain statements, we plan to omit binder and optimizer to keep the code
+    # clean. So, we handle such cases here and pass the statement directly to the
+    # executor.
+    plan_generator = kwargs.pop("plan_generator", PlanGenerator(evadb))
+    if not isinstance(stmt, SKIP_BINDER_AND_OPTIMIZER_STATEMENTS):
+        StatementBinder(StatementBinderContext(evadb.catalog)).bind(stmt)
+        l_plan = StatementToPlanConverter().visit(stmt)
+        p_plan = plan_generator.build(l_plan)
+    else:
+        p_plan = stmt
+    output = PlanExecutor(evadb, p_plan).execute_plan(
+        do_not_raise_exceptions, do_not_print_exceptions
+    )
+    if output:
+        batch_list = list(output)
+        return Batch.concat(batch_list, copy=False)
+
+
 def execute_query(
```

Comment:
What is the difference between `execute_query`, `execute_query_fetch_all` and `execute_statement` now? It seems that they all do the same thing. I think originally `execute_query` is returning a generator, which intends to be used to implement fetch_all, fetch_one, though we never implemented fetch_one.

Reply:
Surpassing this comment as this is breaking the build, and we need to hot fix it.